src/components/TeamGridLayout/index.tsx

```diff
@@ -1,63 +1,48 @@
+import React, { useState } from "react";
 import Grid from "@mui/material/Grid2";
-
-import {
-  Card,
-  CardContent,
-  Typography,
-  Tooltip,
-  IconButton,
-} from "@mui/material";
 import SectionTitle from "../SectionTitle";
-
-import GitHubIcon from "@mui/icons-material/GitHub";
-import EmailIcon from "@mui/icons-material/Email";
-import LinkedInIcon from "@mui/icons-material/LinkedIn";
 import { membersData } from "../../data/membersData";
+import TeamMemberCardFront from "../TeamMemberCardFront/index";
+import TeamMemberCardBack from "../TeamMemberCardBack/index";
+
 
 export default function Team() {
```

Comment:
Decide whether to call it Team or TeamGridLayout

Reply:
got it - will go with TeamGridLayout - as I think we may end up with a team grid layout and a team carousel layout seperately